### ISSUE TITLE
Request denied due to expired auth credentials...

### ISSUE THREAD

**Issue by WangLeeds** [ritiek/scribd-downloader#45]
...but my credentials are still available and I can log in easily. I've just sent you (Ritiek) the credentials so that you can verify it.

**Comment by ritiek** [comment]
Yep, I received your mail. I've got some things to take care of at the moment so I'll probably take a look at this tomorrow.

**Comment by WangLeeds** [comment]
Thank you so much. If you need, I can gift you Scribd account to use for 1 year, so that you can do the coding continuously without lacking of premium account. :)

**Comment by ritiek** [comment]
Lol. Only if you could also sit for my college exams. :D

**Comment by ritiek** [comment]
@WangLeeds I pushed a patch in #47. See if installing the development version and replacing the cookie values fixes it for you now. If it works, I'll publish v1.2.2 on PyPI.

**Comment by WangLeeds** [comment]
It does not work, Ritiek.
Here are the 2 screenshots.
![screen shot 2019-02-13 at 10 40 01 pm](https://user-images.githubusercontent.com/47503428/52763210-89a54400-2fe0-11e9-9bde-423ecca6fb05.png)
![screen shot 2019-02-13 at 10 40 07 pm](https://user-images.githubusercontent.com/47503428/52763212-89a54400-2fe0-11e9-84b5-53c3074aee8c.png)
If you want, I can give you the TeamViewer ID so that you can check this bug easier.
Thanks so much, Ritiek.

**Comment by WangLeeds** [comment]
I've checked many times to ensure I enter the correct session's value.

**Comment by ritiek** [comment]
@WangLeeds That's weird, I just tried it and it seems to work for me. Anyway, #50 shouldn't take much long to land on master branch and we could have then have a release on PyPI soon enough.

**Comment by ritiek** [comment]
I release v1.3.0. See https://github.com/ritiek/scribd-downloader/issues/40.
